### ISSUE TITLE
Robot Vacuum | Incorrect command parameter for suction power level

### ISSUE THREAD

**Issue by a-d-r-i-a-n-h** [hsakoh/switchbot-mqtt#37]
First, thanks for the great add-on! I use it with my Switchbot K10+ by making use of the EnforceDeviceTypes function and defining it as 'DeviceType: Robot Vacuum Cleaner S1'. Thus, I refer to the documentation related to the Robot Vacuum S1.

In addition, in the following description, I replaced the DeviceID with (_number_).

In the Device Configuration the 'PowLevel' command is correctly exposed with the possibility to set following levels: 0 (Quiet), 1 (Standard), 2 (Strong), 3 (MAX). This also complies with the functions of the K10+. In my MQTT integration (core-mosquitto) the 'PowLevel' is displayed as a separate device. There I have two entities:

1. select.powerlevel_4_cmd_(_number_) and
2. button.btn_4_cmd_(_number_)

If I change via the select field e.g. the suction power level from 'Standard' to 'Quiet' I see in the protocol:

`20:23:01 trce: SwitchBotMqttApp.Services.MqttCoreService[0] receive command (_number_),{   "CommandType": "command",   "Command": "PowLevel",   "ParamName": "powerlevel",   "ParamValue": "Quiet" }`

Then I press the button (I guess this is necessary to confirm and send the selected suction power level) I see in the protocol:

```
20:25:46 trce: SwitchBotMqttApp.Services.MqttCoreService[0] receive command (_number_),{   "CommandType": "command",   "Command": "PowLevel",   "ParamName": "btn",   "ParamValue": "action" }
20:25:46 trce: SwitchBotMqttApp.Logics.SwitchBotApiClient[0] Send POST,devices/(_number_)/commands,{"commandType":"command","command":"PowLevel","parameter":"Quiet"}
20:25:47 trce: SwitchBotMqttApp.Logics.SwitchBotApiClient[0] Response POST,https://api.switch-bot.com/v1.1/devices/(_number_)/commands,{"statusCode":190,"body":{},"message":"parameters error"}
20:25:47 fail: SwitchBotMqttApp.Services.MqttCoreService[0] subscribe action (_number_),{   "CommandType": "command",   "Command": "PowLevel",   "ParamName": "btn",   "ParamValue": "action" } System.ApplicationException: {"statusCode":190,"body":{},"message":"parameters error"}    at SwitchBotMqttApp.Logics.SwitchBotApiClient.SendAndEnsureSuccessAsync[TBody](HttpRequestMessage requestMessage, CancellationToken cancellationToken, Int32[] additionalStatus, String bodyForLogging) in C:\github\hsakoh\switchbot-mqtt\src\SwitchBotMqttApp\Logics\SwitchBotApiClient.cs:line 146    at SwitchBotMqttApp.Logics.SwitchBotApiClient.SendDeviceControlCommandAsync(DeviceBase device, CommandConfig command, Object value, CancellationToken cancellationToken) in C:\github\hsakoh\switchbot-mqtt\src\SwitchBotMqttApp\Logics\SwitchBotApiClient.cs:line 193    at SwitchBotMqttApp.Services.MqttCoreService.ReceiveCommandAsync(String payloadRaw, DeviceBase device) in C:\github\hsakoh\switchbot-mqtt\src\SwitchBotMqttApp\Services\MqttCoreService.cs:line 379
```

I would assume that the problem is, that the 'parameter' is 'Quiet' instead of the numeric representation (Quiet = 0).

According to the Switchbot API documentation: https://github.com/OpenWonderLabs/SwitchBotAPI
```
deviceType: Robot Vacuum Cleaner S1
commandType: command
Command: PowLevel
command parameter: {0-3}
Description: set suction power level: 0 (Quiet), 1 (Standard), 2 (Strong), 3 (MAX)
```

When I try it via the Command Test tool of the Addon it works fine:

```
20:34:30 trce: SwitchBotMqttApp.Logics.SwitchBotApiClient[0] Send POST,devices/(_number_)/commands,{"commandType":"command","command":"PowLevel","parameter":"0"}
20:34:33 trce: SwitchBotMqttApp.Logics.SwitchBotApiClient[0] Response POST,https://api.switch-bot.com/v1.1/devices/360TY420703031091/commands,{"statusCode":100,"body":{"items":[{"deviceID":"(_number_)","code":100,"body":{},"message":"OK"}]},"message":"success"}
```

So, preferably, I would like to still have the text in the select-entity (Quiet, Standard, Strong, MAX), but in the 'background' those values need to be translated to the respective numeric value (0, 1, 2, 3) to be correctly set in the "parameter".

**Comment by hsakoh** [comment]
@a-d-r-i-a-n-h 
Thank you for your feedback.
There was an issue where the correct value was not being sent in commands that use a single input parameter with a select box.
I have just fixed it and released version v1.0.22.
I would appreciate your feedback after trying it out.

**Comment by a-d-r-i-a-n-h** [comment]
@hsakoh 
Thank you for the quick fix. Just installed and tried the fixed version and it works. Thank you!
